src/app/views/cheat-sheets/balancers/balancers.component.scss

```diff
@@ -0,0 +1,4 @@
+.balancer-icons>div {
+    display: inline-block;
+    width: 50px

```

Comment:
What are you tying to do here, space them out evenly horizontally?
Why 50px specifically?

Reply:
Looked closer, 50px wasn't right. I have changed it to 38px (icons are 32px, plus 3px * 2 padding on app-factorio-icon).

I was keeping them spaced so that each color lined up: 
![intended](https://user-images.githubusercontent.com/15526875/48661162-94246180-ea22-11e8-9b47-816d167235a8.png) 
Normally, divs look like this:
 ![default](https://user-images.githubusercontent.com/15526875/48661164-ac947c00-ea22-11e8-9145-6fc14db1dba7.png)
Inline elements (default for span) ignore width, and without setting width, inline-block items look like this:
![without width](https://user-images.githubusercontent.com/15526875/48661166-b28a5d00-ea22-11e8-9f18-b1c4b4519ed6.png)

Reply:
Also, I didn't want to have them as separate table columns because I wanted them to be next to each other, and this just seemed the simplest way at the time.